### PR TITLE
timedistancematrix api md

### PR DESCRIPTION
**Pull request by kdiluca**
Initial draft for timedistance api md & renamed elevation to be consistent.  @dnesbitt61

### CODE REVIEW

File: timedistance-api.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+
+# Valhalla Time Distance Service API Reference
+
+The time distance service is a free, open-source web API (and c++ library) that provides a quick computation of time and distance between a set of locations. This page documents the 
+inputs and outputs to the service.
+
+The time distance service is in active development. You can follow the [Mapzen blog](https://mapzen.com/blog) to get updates. To report software issues or suggest enhancements, open an 
+issue in [<?> GitHub repository](https://github.com/valhalla/<?>/issues). You can also send a message to [routing@mapzen.com](mailto:routing@mapzen.com).
```
Comment (kevinkreiser) [inline timedistance-api.md]: this is probably just the thor repository

File: timedistance-api.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+	
+#### API keys and Service Limits
+
+To use the Valhalla routing service, you must first obtain a free developer API key from Mapzen. Sign in at https://mapzen.com/developers to create and manage your API keys.
+
+Valhalla is a free, shared routing service. As such, there are limitations on requests, maximum distances, and numbers of locations to prevent individual users from degrading the overall 
+system performance.
```
Comment (kevinkreiser) [inline timedistance-api.md]: accidental new line

**Comment by kevinkreiser** [comment]
:rocket:

**Comment by rmglennon** [comment]
Let's hold off on renaming the elevation md file. It might break links from the blog.

File: timedistance-api.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+To use the Valhalla routing service, you must first obtain a free developer API key from Mapzen. Sign in at https://mapzen.com/developers to create and manage your API keys.
+
+Valhalla is a free, shared routing service. As such, there are limitations on requests, maximum distances, and numbers of locations to prevent individual users from degrading the overall system performance.
+
+The following time and distance limitations that are currently in place:
+
+* `one_to_many` and `many_to_one` requests have a limit of <?> locations.
+* `many_to_many` requests have a limit of <?> locations.
+* max distance between any two locations is <will these be the same as the limits on the route api reference page?>.
```
Comment (dnesbitt61) [inline timedistance-api.md]: TODO - we need to come up with some total distance limit - all locations are within X miles/km of each other or perhaps some centroid and radius limit. Not sure which is easiest to explain.

**Comment by kdiluca** [comment]
@dnesbitt61 , please also review doc updates for matrix. Thanks!

**Comment by dnesbitt61** [comment]
:shipit:

**Pull request merged**